test/test_data/test_target_validator.py

```diff
@@ -212,6 +213,21 @@ def test_targetvalidator_supported_types_classification(input_data_targettest):
             for col in range(transformed_y.shape[1]):
                 assert np.min(transformed_y[:, col]) == 0
                 assert np.max(transformed_y[:, col]) == len(np.unique(transformed_y[:, col])) - 1
+
+        # Make sure we can perform inverse transform
```

Comment:
Could this be made even stricter by checking the exact values after the inverse transform?

Reply:
Done!